app/steps/ui/documents/index.js

```diff
@@ -21,29 +24,34 @@ class Documents extends ValidationStep {
     runnerOptions(ctx, session) {
         const formdata = session.form;
         const options = {};
+        const documentsWrapper = new DocumentsWrapper(formdata);
+        const documentsRequired = documentsWrapper.documentsRequired(featureToggle.isEnabled(session.featureToggles, 'ft_new_deathcert_flow'));
 
-        if (ctx.caseType === caseTypes.INTESTACY) {
-            const deceasedMarried = Boolean(formdata.deceased && formdata.deceased.maritalStatus === 'optionMarried');
-            const applicantIsChild = Boolean(formdata.applicant && (formdata.applicant.relationshipToDeceased === 'optionChild' || formdata.applicant.relationshipToDeceased === 'optionAdoptedChild'));
-            const noDocumentsUploaded = !(formdata.documents && formdata.documents.uploads && formdata.documents.uploads.length);
-            const iht205Used = Boolean(formdata.iht && formdata.iht.method === 'optionPaper' && formdata.iht.form === 'optionIHT205');
-
-            if (!((deceasedMarried && applicantIsChild) || noDocumentsUploaded || iht205Used)) {
-                options.redirect = true;
-                options.url = '/thank-you';
-            }
+        if (!documentsRequired) {
+            options.redirect = true;
+            options.url = '/thank-you';
         }
-
```

Comment:
If the feature toggle is off, do the old intestacy rules not apply?

Reply:
I added the intestacy condition to the documents wrapper - thought it was neater in there!

Reply:
https://github.com/hmcts/probate-frontend/blob/193527a354a2d9410d9bfe2f52392b6051281c13/app/wrappers/Documents.js#L18

Reply:
cool good move!